lib/inspector/inspector.dart

```diff
@@ -0,0 +1,181 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library inspector;
+
+import 'dart:async';
+
+import 'package:vm_service_lib/vm_service_lib.dart';
+
+import '../framework/framework.dart';
+import '../globals.dart';
+import '../service_extensions.dart' as extensions;
+import '../ui/custom.dart';
+import '../ui/elements.dart';
+import '../ui/icons.dart';
+import '../ui/primer.dart';
+import '../ui/split.dart';
+import '../ui/ui_utils.dart';
+
+import 'inspector_controller.dart';
+import 'inspector_service.dart';
+import 'inspector_tree_canvas.dart';
+import 'inspector_tree_html.dart';
+import 'inspector_tree_web.dart';
+
+// Generally the canvas tree renderer  is a better fit for the inspector.
+// The html renderer is more appropriate for small static trees such as those
+// generated in the logging view.
+bool _useHtmlInspectorTreeRenderer = false;
+
+// TODO(jacobr): add UI to view and configure the pub root directory.
+
+class InspectorScreen extends Screen {
+  InspectorScreen()
+      : super(
+          name: 'Inspector',
+          id: 'inspector',
+          iconClass: 'octicon-device-mobile',
```

Comment:
no trailing comma may make this particular instance format better?

Reply:
Without the trailing comma, dartfmt generates
```dart
  InspectorScreen()
      : super(
            name: 'Inspector',
            id: 'inspector',
            iconClass: 'octicon-device-mobile');
```
which I feel makes the end paren less obvious.
As an aside, I would personally be a fan of setting the line length to 100 for this code base
in case this line would just be a natural one liner.